docs/janssen-server/auth-server/uma-features/claims-gathering-endpoint.md

```
@@ -9,14 +9,25 @@ tags:
-If you have content you'd like to contribute to this page in the meantime, you can get started with our [Contribution guide](https://docs.jans.io/head/CONTRIBUTING/).
+- [Interactive Claims Gathering Flow (Section 3.3.2)](https://docs.kantarainitiative.org/uma/wg/rec-oauth-uma-grant-2.0.html#rfc.section.3.3.2)
+- [Returning Authorization Data to the Client (Section 3.3.3)](https://docs.kantarainitiative.org/uma/wg/rec-oauth-uma-grant-2.0.html#rfc.section.3.3.3)
+
+## Configure Claims Gathering
+
+Claims Gathering is driven by the UMA authorization policies configured for a protected resource and the associated Claims Gathering script.
+
+- To enable or disable the UMA feature, see the [UMA Feature Flag](../../reference/json/feature-flags/janssenauthserver-feature-flags.md#uma).
+- To configure UMA authorization policies, see [UMA RPT Policy](../../../script-catalog/uma_rpt_policy/uma-rpt.md).
+- To implement and configure a Claims Gathering script, see [UMA Claims Gathering (Web Flow)](../../../script-catalog/uma_claims_gathering/uma-claims-web.md).
+- For UMA-related Authorization Server configuration properties, see the [Janssen Authorization Server Configuration Properties](../../reference/json/properties/janssenauthserver-properties.md). For example, the `umaTicketLifetime` property is documented [here](../../reference/json/properties/janssenauthserver-properties.md#umaticketlifetime).
```

markdownlint-cli2 (0.23.2)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)